source/run-java-queries.txt

```diff
@@ -50,8 +50,8 @@ resembles the following:
       )).into(new ArrayList<>());
```

Comment:
I think this one we wanted to specify it as the "run button" because it's not exactly a UI element that says **Run**, but it's an icon, which makes me hesitant to just say "Click Run." 

For now, opted to change the wording to "icon" so it's a bit more specific to what users should be looking for & not mislead them into looking for a literal button that says "Run"